WalletCore/Classes/ViewModel/TransactionsViewModel.swift

```diff
@@ -47,6 +47,9 @@ open class TransactionsViewModel {
     private let transactionModels = Variable<[LWBaseHistoryItemType]>([])
     
     private let transactionsToDisplay = Variable<[LWBaseHistoryItemType]>([])
+    
+    //enable download buttons
+    public var isDownloadButtonEnabled: Observable<Bool>
```

Comment:
Please make this Observable as Driver, so you make sure UI changes will be performed on the UI Thread